### PR TITLE
[DERCBOT-906] Formatting the RAG debug data

### PR DESCRIPTION
**Pull request by assouktim**

### CODE REVIEW

File: gen-ai/orchestrator-server/src/main/python/server/src/gen_ai_orchestrator/services/langchain/callbacks/retriever_json_callback_handler.py [hunk 1]
```diff
@@ -168,15 +169,14 @@ def show_records(self, record_name: str = None):
...
-
-        prompt = (
-            prompt.replace(_prefix_colored_text, '')
-            .replace(_suffix_colored_text, '')
-            .replace(_prompt_after_formatting, '')
-        )
-        return prompt
+
+        # remove ansi escape sequences
+        ansi_escape = re.compile(r'\x1B\[[0-?]*[ -/]*[@-~]')
```
Comment (Benvii) [inline gen-ai/orchestrator-server/src/main/python/server/src/gen_ai_orchestrator/services/langchain/callbacks/retriever_json_callback_handler.py:177]: Any reference for this expression ? maybe add a source as comment

**Review by Benvii — APPROVED** [review]
Ok for me if you can add a reference for the regex expression in comment.
Did you run sonar on it ?

Thanks in advance.

File: gen-ai/orchestrator-server/src/main/python/server/src/gen_ai_orchestrator/services/langchain/callbacks/retriever_json_callback_handler.py (hunk 1 above)
Comment (assouktim) [reply gen-ai/orchestrator-server/src/main/python/server/src/gen_ai_orchestrator/services/langchain/callbacks/retriever_json_callback_handler.py:183]: Yes, the sonar is OK. But no source for regExp.

Comment (Benvii) [reply gen-ai/orchestrator-server/src/main/python/server/src/gen_ai_orchestrator/services/langchain/callbacks/retriever_json_callback_handler.py:183]: Ok never mind for the source if it works ^^

**Pull request merged**